jsapp/js/account/add-ons/oneTimeAddOnRow.component.tsx

```diff
@@ -24,24 +24,48 @@ const quantityOptions = Array.from(
 );
 
 export const OneTimeAddOnRow = ({
-  product,
+  products,
   isDisabled,
   organization,
 }: OneTimeAddOnRowProps) => {
+  const [selectedProduct, setSelectedProduct] = useState(products[0]);
   const [quantity, setQuantity] = useState('1');
-  const [selectedPrice, setSelectedPrice] = useState(product.prices?.[0]);
+  const [selectedPrice, setSelectedPrice] = useState<Product['prices'][0]>(
+    selectedProduct.prices[0]
+  );
   const displayPrice = useDisplayPrice(selectedPrice, parseInt(quantity));
   const priceOptions: KoboSelectOption[] = useMemo(
     () =>
-      product.prices.map((price) => {
-        return {value: price.id, label: product.name};
+      selectedProduct.prices.map((price) => {
+        return {value: price.id, label: price.recurring?.interval || 'me'};
       }),
-    [product]
+    [selectedProduct]
   );
 
+  let displayName;
+  let description;
+
+  if (selectedProduct.name.includes('ASR' && 'MT')) {
+    displayName = 'NLP Package';
+    description =
+      'Increase your transcription minutes and translations characters.';
```

Comment:
Make sure to wrap any strings that will be shown to the user in `t()` so that they can be translated.